kits/sparktrace/apps/.env.example

```
@@ -0,0 +1,55 @@
+# analyst        -> Claude Haiku 4.5  (claude-haiku-4-5)
+# reporter       -> Claude Sonnet 5   (claude-sonnet-5)
+SPARKTRACE_PLANNER_FLOW_ID="SPARKTRACE_PLANNER_FLOW_ID"
+SPARKTRACE_REPO_READER_FLOW_ID="SPARKTRACE_REPO_READER_FLOW_ID"
+SPARKTRACE_QUERY_GEN_FLOW_ID="SPARKTRACE_QUERY_GEN_FLOW_ID"
+SPARKTRACE_ANALYST_FLOW_ID="SPARKTRACE_ANALYST_FLOW_ID"
+SPARKTRACE_REPORTER_FLOW_ID="SPARKTRACE_REPORTER_FLOW_ID"
+
+# ── AWS credentials ─────────────────────────────────────────
+# Used by apps/lib/aws/{athena,glue,s3}-client.ts. Prefer an
+# assumed role / short-lived credentials over long-lived keys.
+AWS_ACCESS_KEY_ID="AWS_ACCESS_KEY_ID"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)